internal/test/framework/framework.go

```diff
@@ -202,18 +202,26 @@ func (f *TestFramework) getOpenShiftOperatorClient(config *restclient.Config) er
 // are running against. For example: the kubernetes API server endpoint https://api.abc.devcluster.openshift.com:6443
 // gets converted to abc.devcluster.openshift.com
 func (f *TestFramework) getClusterAddress(config *restclient.Config) error {
```

Comment:
I don't follow what you mean by `this commit changes the WMCB to use internal ip instead of external ip
for retrieving the node.` We are using the API server internal endpoint and the private IP of the VM.

Reply:
changed the commit message to make it more clear